alarmpresenter/main.qml

```diff
@@ -169,7 +169,7 @@ Application {
         onTriggered: {
             feedback.stop()
             if(alarmDialog !== undefined && alarmDialog !== null)
-                alarmDialog.snooze()
```

Comment:
Line 190 there's a similar use of snooze() on powerkey presses (to be fair I have no memory of that code and I don't think I ever used it, if it doesn't work we might as well scrap it off) - if it works, this should probably also be a close() and possibly a shared function

Reply:
I see, that makes sense yes. Thank you!